cli/azd/test/functional/telemetry_test.go

```diff
@@ -299,7 +299,8 @@ func Test_CLI_Telemetry_NestedCommands(t *testing.T) {
 			require.Contains(t, m, fields.CmdEntry)
 			require.Equal(t, "cmd.up", m[fields.CmdEntry])
 
-			require.NotContains(t, m, fields.CmdFlags)
+			require.Contains(t, m, fields.CmdFlags)
```

Comment:
Guessing the fact that the flags are present for up is now because the command is less "special" before?

Reply:
In the case of the default workflow i have specified that `package` and `deploy` to be called with `--all` flag which ensures we get same behavior as before which then forces the CmdFlags to have these flags.